src/tools/spin/spinquic.cpp

```diff
@@ -202,7 +202,9 @@ class SpinQuicWatchdog {
 };
 
 static QUIC_API_TABLE MsQuic;
-// This locks MsQuicOpen2 in RunThread when statically linked with libmsquic
+//
+// This lock is used to protect concurrent MsQuic library open and close.
+//
 CXPLAT_LOCK RunThreadLock;
```

Comment:
Was the intention of `RunThreadLock` always to protect concurrent accesses to MsQuic library open/close?
If so, maybe a better name would be "MsQuicLibLock" or something?

Reply:
Yeah, that's a better name. Let me see if this PR fixes the problem first.

Reply:
I think the intention was to protect it only if doing static link (because dynamic link MUST support parallel callers in the same process). I'm fine with changing the name though.